src/main/java/org/betonquest/betonquest/quest/condition/DoNothingPlayerlessCondition.java

```diff
@@ -0,0 +1,21 @@
+package org.betonquest.betonquest.quest.condition;
+
+import org.betonquest.betonquest.api.quest.condition.PlayerlessCondition;
+import org.betonquest.betonquest.exceptions.QuestRuntimeException;
+
+/**
+ * A playerless condition placeholder that is always false.
+ */
+public class DoNothingPlayerlessCondition implements PlayerlessCondition {
+
+    /**
+     * Create a playerless condition that does nothing and is always false.
+     */
+    public DoNothingPlayerlessCondition() {
+    }
+
+    @Override
+    public boolean check() throws QuestRuntimeException {
+        throw new QuestRuntimeException("This condition cannot be checked in the current context.");
```

Comment:
I personally prefer this solution by throwing an exception, but I am unsure if this is actually good, as this effects a lot of BQ behaviour. We need to discuss this before merging this PR

Reply:
I also like this pattern more because the prior existing class was already something to fill the gap of using a event in a broken state which is hard to detect when just nothing happens.

Also, the classes need a better name now.